src/App.tsx

```diff
@@ -1,14 +1,57 @@
-import React from 'react';
+import React, { useCallback, useMemo, useState } from 'react';
 import './App.scss';
+import { debounce } from 'lodash';
 import { peopleFromServer } from './data/people';
+import { DropdownItem } from './components/DropdownItem';
+import { Person } from './types/Person';
 
 export const App: React.FC = () => {
-  const { name, born, died } = peopleFromServer[0];
+  const [visiblePeople] = useState(peopleFromServer);
+
+  const [query, setQuery] = useState('');
+  const [appliedQuery, setAppliedQuery] = useState('');
+
+  const [isHideList, setIsHideList] = useState(true);
```

Comment:
hide list - приховати список, а ти хочеш сказати "чи список прихований", тому порядок слів буде трохи інакший:
```suggestion
  const [isListHided, setIsListHided] = useState(true);
```